js/viewer/annotation/annotation-group.js

```diff
@@ -57,6 +71,57 @@ function AnnotationGroup(id, anatomy, description, parent){
         return annotation;
     }
 
+    // this.addMarkerDef = function (stroke, subtype, checkExists = true) {
```

Comment:
Why did you leave this here? You should add a jsdoc style of comment to this function. Take a look at other functions as an example.

Reply:
Removed the comment and added a JSDoc comment.